packages/block-library/src/template-part/edit/index.js

```diff
@@ -247,10 +285,36 @@ export default function TemplatePartEdit( {
 									} )
 								}
 							>
-								{ __( 'Edit' ) }
+								{ hasNavigationBlocks
+									? sprintf(
+											/* translators: %s: template part title or fallback text */
+											__( 'Edit %s' ),
+											(
+												title || __( 'template part' )
+											).toLowerCase()
+									  )
+									: __( 'Edit' ) }
 							</ToolbarButton>
 						</BlockControls>
 					) }
+				{ hasNavigationBlocks && (
+					<BlockControls group="other">
+						<ToolbarButton
+							label={ __( 'Edit navigation' ) }
+							onClick={ () => {
+								// Select the first Navigation block
+								selectBlock( firstNavigationBlockId );
+								// Enable the complementary area (inspector)
+								enableComplementaryArea(
+									'core',
+									'edit-post/block'
```

Comment:
Is `edit-post/block` always the right complimentary area?

Reply:
Yeh I'm not sure on that one 🤔